### DISCUSSION TITLE
Add option for "Start of day time"

### DISCUSSION THREAD

**Discussion by 97997** [iSoron/uhabits#881] [Feature Requests]
### Update: 

I just realized this feature has been suggested before in https://github.com/iSoron/uhabits/pull/621#issue-468494937 sorry for opening this useless discussion

### Original post text:

Allow users to change the time of the day when a new day starts (it's 00:00 by default) since most people don't go to sleep exactly at midnight.

I would really love to have this feature, since I use the habit check widgets and usually i go to sleep at very late hours like 5am

**Comment by Mar2ck** [comment]
I don't think this discussion is useless, despite what iSolon says in that thread 3pm doesn't work for everyone. I find it confusing why an option like this would be hardcoded. 

It would work much better for people like me if theres was a hour/time select widget so i could choose what works for me. AnkiDroid for example just have a slider from 0 to 23 that let's you select the hour. It doesn't have to be as granular aa picking the exact minute, it just has to be customizable.
